test_setup.unmanage.yml

```diff
@@ -0,0 +1,81 @@
+---
+# This is an example of a ansible playbook to uninstall Tendrl according to Red Hat Gluster Storage Monitoring Guide
+# https://access.redhat.com/documentation/en-us/red_hat_gluster_storage/3.3/html/monitoring_guide/cluster_expansion#unmanaging_cluster
+#
+#
+# The playbook expects inventory file with the following groups:
+#
+# * tendrl-server: a single machine where Tendrl Server (management console) is
+#                  installed
+# * gluster-servers: gluster storage servers (machines with bricks)
+
+
+# Stop individual services on the Storage Nodes
+- hosts: gluster-servers
```

Comment:
We should use `gluster_servers` group instead. The name was changed this way in tendrl-ansible to prevent problems with future version of ansible.